### ISSUE TITLE
Unable to hash header file

### ISSUE THREAD

**Issue by joestringer** [cilium/cilium#3807]
On startup, I have seen messages like this:

```
Apr 19 06:10:08 runtime1 cilium-agent[7293]: level=warning msg="Unable to hash header file" containerID=96ae9f244a endpointID=29381 error="bufio.Scanner: token too long" policyRevision=16
```

Is this harmless?

**Comment by ianvernon** [comment]
I believe this should be fixed by https://github.com/cilium/cilium/pull/3798 .

**Comment by rlenglet** [comment]
Those messages are harmless. But they were a symptom that we were potentially missing some optimizations.
Fixed in #3798
